wildfly-subsystem-archetype/pom.xml

```diff
@@ -145,27 +144,11 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>org.wildfly.core</groupId>
-                <artifactId>wildfly-controller</artifactId>
-                <version>${version.wildfly.core}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.wildfly.core</groupId>
-                <artifactId>wildfly-server</artifactId>
-                <version>${version.wildfly.core}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.wildfly.core</groupId>
-                <artifactId>wildfly-subsystem-test</artifactId>
+                <groupId>org.wildfly</groupId>
+                <artifactId>wildfly-parent</artifactId>
                 <type>pom</type>
-                <scope>test</scope>
-                <version>${version.wildfly.core}</version>
-            </dependency>
-            <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
-                <scope>test</scope>
-                <version>${version.junit}</version>
+                <version>${version.wildfly}</version>
+                <scope>import</scope>
```

Comment:
sure, but with this you get almost 300 components as dependency which bit huge.
In most cases even having all wildfly-core deps would be too much, but would make more sense.